tests/locator_test.go

```diff
@@ -45,10 +45,36 @@ func TestLocatorDblclick(t *testing.T) {
 		link.Dblclick(nil)
 		require.True(t, dblclick(), "could not double click the link")
 	})
-	// There are two links in the document (strict_link.html).
+	// There are two links in the document (locators.html).
 	// The strict mode should disallow selecting multiple elements.
 	t.Run("strict", func(t *testing.T) {
 		link := p.Locator("a", nil)
 		require.Panics(t, func() { link.Dblclick(nil) })
 	})
 }
+
+func TestLocatorCheck(t *testing.T) {
+	tb := newTestBrowser(t, withFileServer())
+	p := tb.NewPage(nil)
+	require.NotNil(t, p.Goto(tb.staticURL("/locators.html"), nil))
+
+	// Selecting a single element and checking it is OK.
+	t.Run("ok", func(t *testing.T) {
+		check := func() bool {
+			cr := p.Evaluate(tb.toGojaValue(`() => window.check`))
+			return cr.(goja.Value).ToBoolean() //nolint:forcetypeassert
```

Comment:
To get rid of the `nolint` (same for other tests in this file):
```suggestion
			cb, ok := cr.(goja.Value)
			require.True(t, ok)
			return cb.ToBoolean()
```

Reply:
Alright, I'll address this in one Go for all tests after merging `Check` and `Uncheck`.